clang-tools-extra/clangd/support/Markup.h

```diff
@@ -37,6 +37,14 @@ class Block {
 
   virtual bool isRuler() const { return false; }
   virtual ~Block() = default;
+
+protected:
```

Comment:
If the field is `Paragraph`-specific, should we just move it into that class and make it private?